src/systems/material.js

```diff
@@ -195,6 +205,12 @@ module.exports.System = registerSystem('material', {
   }
 });
 
+function isHLS (videoEl) {
+  if (videoEl.type === 'application/x-mpegurl') return true;
```

Comment:
do MIME types get lowercased automatically when calling the `type` getter? 

I notice in [Apple's docs](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/StreamingMediaGuide/DeployingHTTPLiveStreaming/DeployingHTTPLiveStreaming.html), it appears as `application/x-mpegURL`.

- also, do you want to check `vnd.apple.mpegURL` as well?
- and what about `video/MP2T` (`.ts`)?